client/src/services/ApiService.js

```diff
@@ -136,7 +136,7 @@ class ApiService {
       const isFinalOp = i === operations.length - 1;
       const basePropertyName = op === REL_TYPE_INCOMING ? "$targetId" : "$sourceId";
       let count = 1;
-      const twinIdsList = twinIds.map(twinId => `'${twinId}'`);
+      const twinIdsList = twinIds.map(twinId => `'${twinId.replaceAll("'", "\\'")}'`);
```

Comment:
As per above. I'm not sure if the goal here is to remove this. The quote is a valid part of the twin name, we just need to be able to manage it.

Reply:
Whit this specific change we are not removing the quote. This map is actually creating an SQL Inyection that is failing because in the example you have a node called B' the inyection is truncated as 'Select * ... A, B', C'
This change just fix the inyection like this 'Select * ... A, B\', C' because its actually just inserting a \ not removing the quote

Reply:
Oops - I completely misread this line. Perfect!